core/server/data/import/utils.js

```diff
@@ -234,6 +234,11 @@ utils = {
                 post.created_at = Date.now();
             }
 
+            _.each(['created_at', 'updated_at', 'published_at'], function (fieldName) {
```

Comment:
I mentioned that in my previous PR, will gladly adjust this to be more generic. Can have a look later.